zeekclient/__init__.py

```diff
@@ -0,0 +1,23 @@
+import logging
```

Comment:
I don't see `logging` getting used in this file, so this import might not be necessary.

Reply:
You're faster than my upcoming pylinting. :-) Thanks!